### ISSUE TITLE
"Ordinal" vs "Cardinal" on pypi

### ISSUE THREAD

**Issue by NotTheEconomist** [savoirfairelinux/num2words#164]
The *usage* section of the [Pypi project page](https://pypi.python.org/pypi/num2words) lists the result of `num2words(42, to='cardinal')` as the ordinal result

### Expected Behaviour

Text should read:

```
>>> num2words(42, to='cardinal')
forty-two
```

or

```
>>> num2words(42, to='ordinal')
forty-second
```

### Actual Behaviour

Text reads

```
>>> num2words(42, to='cardinal')
forty-second
```

**Comment by NotTheEconomist** [comment]
It looks like this was fixed in the readme in [PR#145](https://github.com/savoirfairelinux/num2words/pull/145) but never got updated on Pypi.

**Comment by NotTheEconomist** [comment]
Just sweeping through old outstanding issues and this appears to have been fixed somewhere along the line.
